examples/cpp/tf_label_image_utils.cc

```diff
@@ -300,6 +300,8 @@ Status GetTopLabels(const std::vector<Tensor>& outputs, int how_many_labels,
 //-----------------------------------------------------------------------------
 Status PrintTopLabels(const std::vector<Tensor>& outputs,
                       const string& labels_file_name) {
+  // Disable nGraph so that we run these using TensorFlow on CPU
+  tensorflow::ngraph_bridge::config::ngraph_disable();
```

Comment:
Why disable the bridge?

Reply:
The bridge is disabled here to run validation using vanilla TF (CPU).